spec/ttml1.xml

```diff
@@ -4594,26 +4594,26 @@ as a style property only to those element types indicated in the following table
 </tr>
 </tbody>
 </table>
-<p>If the value of this attribute is <code>normal</code>, then the
```

Comment:
Looking at the [CSS definition](https://www.w3.org/TR/REC-CSS2/visudet.html#line-height), it is clear that `normal` is treated as ` <number>`, and that when `<number>` is specified, the _number_ not the computed value is inherited. This is not clear either in XSL or in the table above. I think we need a specific line of text to clarify it, e.g.:

> When the specified value is "normal" it is the specified value that is inherited, not the computed value.